### ISSUE TITLE
Set up continuous integration

### ISSUE THREAD

**Issue by dguo** [SynapseFI/SynapseNode#52]
I would have created a PR for #49, but there's no CI for this repo, so I was concerned about breaking something.

It seems that the test command actually sends requests, but I don't want to use our actual client id and secret.

**Comment by jenstroeger** [comment]
Agreed, we need to run at least [eslint](https://eslint.org/) and start adding unit tests.
